benchmark/bm_machine.cpp

```diff
@@ -0,0 +1,89 @@
+//
+// Created by Jelle Spijker on 9/4/20.
+//
+
+#include <benchmark/benchmark.h>
+
+#include <string_view>
+#include <iostream>
+
+#include "GHermeneus/GHermeneus.h"
+
+static const std::string simple_file{"../resources/simple.gcode"};
```

Comment:
Why hardcode this? Isn't it better to make this dynamically settable (eg; file to run it on as a parameter)?

Reply:
absolutely true.
But at the time of writing the unit and benchmark test aren't real CI tests yet. The're intended to be. For now I will leave it and add a ticket to the backlog that we create some proper tests.

Reply:
wont fix